packages/dbml-core/src/export/SqliteExporter.js

```diff
@@ -0,0 +1,128 @@
+import _ from 'lodash';
+import {
+  hasWhiteSpace,
+  shouldPrintSchema,
+  buildJunctionFields1,
+  buildJunctionFields2,
+  buildNewTableName,
+} from './utils';
+import { DEFAULT_SCHEMA_NAME } from '../model_structure/config';
+
+class SQLiteExporter {
+  static exportEnums (enumIds, model) {
+    return [];
+  }
+
+  static getFieldLines (tableId, model) {
+    const table = model.tables[tableId];
+
+    const lines = table.fieldIds.map((fieldId) => {
+      const field = model.fields[fieldId];
+
+      let line = '';
+      if (field.increment) {
+        line = `"${field.name}" INTEGER PRIMARY KEY AUTOINCREMENT`;
+      } else {
+        let typeName = field.type.type_name.toUpperCase();
+        if (typeName === 'SERIAL') {
+          typeName = 'INTEGER';
+        } else if (typeName === 'VARCHAR') {
+          typeName = 'TEXT';
+        }
+        line = `"${field.name}" ${typeName}`;
+      }
+
+      if (field.unique) {
+        line += ' UNIQUE';
+      }
+      if (field.pk && !field.increment) {
+        line += ' PRIMARY KEY';
+      }
+      if (field.not_null) {
+        line += ' NOT NULL';
+      }
+      if (field.dbdefault) {
+        if (field.dbdefault.type === 'expression') {
+          if (field.dbdefault.value === 'now()') {
+            line += ' DEFAULT CURRENT_TIMESTAMP';
+          } else {
+            line += ` DEFAULT (${field.dbdefault.value})`;
+          }
+        } else if (field.dbdefault.type === 'string') {
+          line += ` DEFAULT '${field.dbdefault.value}'`;
+        } else {
+          line += ` DEFAULT ${field.dbdefault.value}`;
+        }
+      }
+
+      return line;
+    });
+
+    return lines;
+  }
+
+  static getTableContentArr (tableIds, model) {
+    const tableContentArr = tableIds.map((tableId) => {
+      const fieldContents = SQLiteExporter.getFieldLines(tableId, model);
```

Comment:
This function needs to export the composite primary keys. You can refer to https://github.com/holistics/dbml/blob/6fb58f32eda8c1a814747510d564cde6abc71cf5/packages/dbml-core/src/export/PostgresExporter.js#L112 to implement it